tests/pytests/unit/modules/test_win_event.py

```diff
@@ -0,0 +1,134 @@
+import datetime
+
+import pytest
+
+import salt.modules.win_event as win_event
+
+pytestmark = [
+    pytest.mark.windows_whitelisted,
+    pytest.mark.skip_unless_on_windows,
+    pytest.mark.destructive_test,
+]
+
+
+@pytest.fixture(scope="function")
+def application_events():
+    win_event.clear("Application")
```

Comment:
Please add a comment this delete the contents of the event log.  Does this need to be run with Admin access?

Reply:
Done